src/main/java/seedu/address/ui/StatusBarFooter.java

```diff
@@ -50,6 +54,7 @@ public StatusBarFooter(String saveLocation, int totalPersons) {
         setSaveLocation("./" + saveLocation);
```

Comment:
What's the difference between the clock in `syncStatus` and the clock in `staticClock`?